modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -78,6 +78,37 @@ module FindPoaByParticipantId
         end
       end
 
+      ##
+      # CorporateUpdateServiceBean
+      #
+      module CorporateUpdateServiceBean
+        DEFINITION =
+          Bean.new(
+            path: 'CorporateUpdateServiceBean',
+            namespaces: Namespaces.new(
+              target: 'http://services.share.benefits.vba.va.gov/',
+              data: nil
+            )
+          )
+      end
+
+      module CorporateUpdateWebService
+        DEFINITION =
+          Service.new(
+            bean: CorporateUpdateServiceBean::DEFINITION,
+            path: 'CorporateUpdateWebService'
+          )
+
+        module UpdatePoaAccess
+          DEFINITION =
+            Action.new(
+              service: CorporateUpdateWebService::DEFINITION,
+              name: 'updatePoaAccess',
+              key: 'return'
+            )
+        end
```

Comment:
I _think_ the goal is to [remove methods](https://jira.devops.va.gov/browse/API-42308) from the definitions file, but it might be better to do it all in one PR 🤷.

Reply:
Yeah gonna hold off on that for the cleanup ticket.